bin/metrics-es-cluster.rb

```diff
@@ -191,7 +191,7 @@ def run
       end
       acquire_cluster_metrics.each do |cluster_metric|
         cluster_metric[1].each do |k, v|
-          output(config[:scheme] + '.' + cluster_metric[0] + '.' + k, v)
+          output(config[:scheme] + '.' + cluster_metric[0] + '.' + k, v || 0)
```

Comment:
Yes, it does, see issue #49. Note: technically _carbon_ errors, but carbon is a component of graphite.

I don't believe the parentheses would make any difference in functionality, happy to add them if its a style thing.